packages/mjml-raw/README.md

```diff
@@ -19,19 +19,3 @@ If placed inside `<mj-head>`, its content will be added at the end of the `<head
   </a>
 </p>
 
-
```

Comment:
this should stay

Reply:
No action. This second blank line is absent in `9d`. It is present in `next`. Perhaps someone else corrected this?

Reply:
I meant the whole block after that not the blank line :)

Reply:
Bump on this block, this is an important thing to keep for 4.4